### PR TITLE
add tests, clean docs

### PR DESCRIPTION
**Pull request by SamDuffield**
Add more sophisticated tests and clean the doc string for `tk_to_quax`

### CODE REVIEW

File: tests/tests_tket.py [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+from jax import numpy as jnp, jit, grad, random
+import qujax
+
+from pytket.circuit import Circuit, Qubit
+from pytket.pauli import Pauli, QubitPauliString
+from pytket.utils import QubitPauliOperator
+from pytket.extensions.qujax import tk_to_qujax
+
+
+def _test_circuit(circuit, param):
```
Comment (cqc-melf) [inline tests/tests_tket.py:24]: This could be the problem?

File: tests/test_tket.py [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+from jax import numpy as jnp, jit, grad, random
+import qujax
+
+from pytket.circuit import Circuit, Qubit
+from pytket.pauli import Pauli, QubitPauliString
+from pytket.utils import QubitPauliOperator
+from pytket.extensions.qujax import tk_to_qujax
+
+
+def _test_circuit(circuit, param):
```
Comment (cqc-melf) [inline tests/test_tket.py:24]: Could you add type hints in this function?

**Review by cqc-melf — CHANGES_REQUESTED** [review]

File: tests/test_tket.py (hunk 2 above)
Comment (SamDuffield) [reply tests/test_tket.py:24]: good call!

File: tests/tests_tket.py (hunk 1 above)
Comment (SamDuffield) [reply tests/tests_tket.py]: problem was file needed to be test_ rather than tests_

**Review by cqc-melf — APPROVED** [review]

**Pull request merged**